packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -212,24 +220,26 @@ export const VBtn = defineComponent({
                 },
               }}
             >
-              <span class="v-btn__append">
-                { slots.append?.() ?? (<VIcon />) }
-              </span>
+              <span class="v-btn__append">{slots.append?.() ?? <VIcon />}</span>
             </VDefaultsProvider>
-          ) }
+          )}
 
-          { !!props.loading && (
+          {!!props.loading && (
             <span key="loader" class="v-btn__loader">
-              { slots.loader?.() ?? (
+              {slots.loader?.() ?? (
```

Comment:
wasn't clear from the issue open whether the `value` prop should actually be displayed in place of the `slot`.

I don't think that would be the correct behaviour as the `value` on button seems to only be useful when passed via a Group?